gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/lru/NewLRUClockHand.java

```diff
@@ -260,9 +275,22 @@ public LRUClockNode getLRUEntry() {
         }
```

Comment:
I think this check can also can be removed. It was also done to avoid similar deadlock. Fine for now.

Reply:
Ok will remove in the next cycle.